tests/utils/devServer.js

```diff
@@ -0,0 +1,92 @@
+const cliPath = require('./cliPath')
+const path = require('path')
+const getPort = require('get-port')
+const seedrandom = require('seedrandom')
+const execa = require('execa')
+const pidtree = require('pidtree')
+
+// each process gets a starting port based on the pid
+const rng = seedrandom(`${process.pid}`)
+function getRandomPortStart(rng) {
+  const startPort = Math.floor(rng() * 10000) + 10000 // 10000 to avoid collisions with frameworks ports
+  return startPort
+}
+
+let currentPort = getRandomPortStart(rng)
+
+const startServer = async ({ cwd, env = {} }) => {
+  const tryPort = currentPort++
+  const port = await getPort({ port: tryPort })
+  const host = 'localhost'
+  const url = `http://${host}:${port}`
+  console.log(`Starting dev server on port: ${port} in directory ${path.basename(cwd)}`)
+  const ps = execa(cliPath, ['dev', '-p', port], {
+    cwd,
+    env: { ...process.env, ...env },
```

Comment:
```suggestion
    env,
```

Execa appends to `env` by default (instead of overwriting it), as opposed to `child_process.spawn()`.

Reply:
Cool, I'll fix it